examples/Makefile.am

```diff
@@ -49,7 +48,6 @@ endif
 
 YAMI_DECODE_LIBS = \
 	$(YAMI_COMMON_LIBS) \
-	$(top_builddir)/decoder/libyami_decoder.la \
 	$(NULL)
 
 if BUILD_STATIC
```

Comment:
Please note, however, that for each bin_PROGRAM you'll want the `-Wl,--whole-archive $(top_builddir)/.libs/libyami.la -Wl,--no-whole-archive` to show up only once per link step, otherwise it'll fail to compile.  That is, watch out for some of the LDFLAGS that build upon others.